crm_action/__openerp__.py

```diff
@@ -5,7 +5,9 @@
 {
     'name': 'CRM Action',
     'version': '8.0.1.2.0',
-    'author': 'Savoir-faire Linux,Odoo Community Association (OCA)',
+    'author': """Savoir-faire Linux,
+                 Kiko Peiro (www.factorlibre.com),
+                 Odoo Community Association (OCA)""",
```

Comment:
This simple patch doesn't grant you to be in the coauthor list. Please remove you from here and add it to contributors in README.rst file.